code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -153,7 +153,7 @@
 		skipcatch = TRUE
 		blocked = TRUE
 	else if(I)
-		if((I.throw_speed >= EMBED_THROWSPEED_THRESHOLD) || I.embedding.embedded_ignore_throwspeed_threshold)
+		if((I.throw_speed >= EMBED_THROWSPEED_THRESHOLD) || I.embedding.embedded_ignore_throwspeed_threshold || I.item_flags & SHRAPNEL)

```

Comment:
Give me good reasons why we're using inherent throw speed in this check or i'll be changing the fix to that.

Reply:
no idea

Reply:
I guess i'll be changing to this, if there's some common rockethrows we'll see.